qa/tasks/cephfs/caps_helper.py

```diff
@@ -152,16 +152,16 @@ def run_mds_cap_tests(self, perm, mntpt=None):
             self.test_set = [(x, y.replace(mntpt, ''), z) for x, y, z in \
                              self.test_set]
 
-        self.conduct_pos_test_for_read_caps()
+        self.run_pos_test_for_read_caps()
```

Comment:
This commit says "Give these this methods a smaller name"

Does this change really help in any way?

Reply:
Yes, it would  more helpful if it could be contracted further.

Reply:
Umm, true. I should remove this change I guess...

We usually name our methods as `run_abcd()`. At least this change aligns this method name with general manner of naming. If this doesn't seem like a good  reason, I'll remove this change. :)

Reply:
sure